test/tune/example_scheduler.py

```diff
@@ -0,0 +1,106 @@
+from functools import partial
+import time
+
+
+def evaluation_fn(step, width, height):
+    return (0.1 + width * step / 100) ** (-1) + height * 0.1
+
+
+def easy_objective(use_raytune, config):
+    if use_raytune:
+        from ray import tune
+    else:
+        from flaml import tune
+    # Hyperparameters
+    width, height = config["width"], config["height"]
+
+    for step in range(config["steps"]):
+        # Iterative training function - can be any arbitrary training procedure
+        intermediate_score = evaluation_fn(step, width, height)
+        # Feed the score back back to Tune.
+        trial_not_finished = tune.report(iterations=step, mean_loss=intermediate_score)
+        if trial_not_finished is False:
+            return
+
+
+def test_tune_scheduler(smoke_test=True, use_ray=True, use_raytune=False):
+    import numpy as np
+    from flaml.searcher.blendsearch import BlendSearch
+
+    np.random.seed(100)
```

Comment:
what for?

Reply:
Was trying to check the consistency between 'flaml.tune' and 'ray.tune'

Reply:
I don't think their consistency relies on np.random.seed(). Could you confirm that?